CHANGELOG.md

```diff
@@ -101,7 +101,6 @@
 - Update etcd-client requirement from 0.10 to 0.11 [#845](https://github.com/apache/arrow-ballista/pull/845) (dependabot[bot])
 - Update hashbrown requirement from 0.13 to 0.14 [#846](https://github.com/apache/arrow-ballista/pull/846) (dependabot[bot])
 - Bump word-wrap from 1.2.3 to 1.2.4 in /ballista/scheduler/ui [#849](https://github.com/apache/arrow-ballista/pull/849) (dependabot[bot])
-- Update hdfs requirement from 0.1.1 to 0.1.4 [#856](https://github.com/apache/arrow-ballista/pull/856) (yahoNanJing)
```

Comment:
The changelog is a record of previous work. We shouldn't remove the history of prior work on HDFS support.

Reply:
my bad, fixed